images/dockerregistry/config.yml

```diff
@@ -35,3 +35,8 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
+  version: 1.0
+  metrics:
+    enabled: false
+    secret: TopSecretToken
```

Comment:
More examples:
```
storage:
  azure:
    accountname: accountname
    accountkey: base64encodedaccountkey
  s3:
    accesskey: awsaccesskey
    secretkey: awssecretkey
  swift:
    username: username
    password: password
  oss:
    accesskeyid: accesskeyid
    accesskeysecret: accesskeysecret
health:
  http:
      headers:
        Authorization: [Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==]
proxy:
  username: username
  password: password
```
@smarterclayton @mfojtik This config file is full of plaintext passwords even if we can fix this. We have already discussed the possibility of using secrets for this file and came to the conclusion that if this is done, then the entire configuration will have to be put there.

> agree, this is abad practice

Don't tell me. In the beginning, I had not even considered this decision. You asked for a shared secret.